FilmwebCloneFrontend/src/types/UserRoleType.ts

```diff
@@ -0,0 +1 @@
+export type UserRoleType = 'Admin' | 'User'
```

Comment:
To już popraw na przyszłość: export type UserRoleType = 'Guest' | 'User' | 'Administrator'